ironfish/src/wallet/wallet.ts

```diff
@@ -634,14 +634,14 @@ export class Wallet {
   ): Promise<Transaction> {
     const heaviestHead = this.chain.head
     if (heaviestHead === null) {
-      throw new ValidationError('You must have a genesis block to create a transaction')
+      throw new Error('You must have a genesis block to create a transaction')
```

Comment:
Do we lose anything in error handling in the RPC endpoints by changing from ValidationError to Error?